### PR TITLE
Ms/new text shadow

### PR DESCRIPTION
**Pull request by martensievers**
When adding text-shadow to the view, I first check if a single string as text_shadow attribute exists to support the newly added text_shadow property from the Incito format, before making sure to render existing text shadows to support backwards compatibility.

### CODE REVIEW

File: lib/coffeescript/views/text.coffee [hunk 1]
```diff
@@ -66,7 +66,9 @@ export default class TextView extends View
         # Text shadow.
         textShadow = @getTextShadow()
```
Comment (mortenbo) [inline lib/coffeescript/views/text.coffee:67]: What about this line?

File: .idea/workspace.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectId" id="1at6GbNNng2Brbsdcw2MMIMFR5A" />
```
Comment (mortenbo) [inline .idea/workspace.xml]: Can you add .idea/ to .gitignore?

Comment (mortenbo) [reply .idea/workspace.xml]: Or git rm as I can see it's already there.

Comment (martensievers) [reply .idea/workspace.xml]: Sure, will do.

File: lib/coffeescript/views/text.coffee (hunk 1 above)
Comment (martensievers) [reply lib/coffeescript/views/text.coffee:67]: I think we still need to check for the old text_shadow to make sure this doesn't break backwards, so I left in the old way and use it as a fallback.

**Comment by martensievers** [comment]
The new text-shadow property adds support for composite shadows by concatenating text-shadows in x,y,r,hex values separated by commas

**Pull request merged**